docs/_docs/collections.md

```diff
@@ -51,7 +51,7 @@ defaults:
 
 Create a corresponding folder (for example, `<source>/_my_collection`) and add
 documents. YAML Front Matter is read in as data if it exists, and everything
-after it is available in the document's `content` attribute. If no YAML Front
+after it is accessible via the document's `content` attribute. If no YAML Front
```

Comment:
see how many times you repeated `front matter`..? `it` was used as a preposition for the same `front matter`.